release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -834,6 +851,21 @@ With this release, the image-based upgrade introduces the following enhancements
 
 For more information about the `ImageBasedGroupUpgrade` API, see xref:../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-concept_ztp-gitops[Managing the image-based upgrade at scale using the ImageBasedGroupUpgrade CR on the hub].
 
+[id="ocp-4-17-disk-encryption_{context}"]
+==== Enabling disk encryption with TPM and PCR protection (Technology Preview)
```

Comment:
I noticed this has the same title as the xref below. Considering this is more reference than procedure, maybe consider changing the title to "About disk encryption.." ?

Reply:
Updated as per the suggestion.